pyfar/dsp/dsp.py

```diff
@@ -2183,9 +2206,13 @@ def normalize(signal, reference_method='max', domain='time',
             input_data = np.abs(signal.freq)
     # get values for normalization max or mean
         if reference_method == 'max':
-            reference = np.max(input_data[..., limits[0]:limits[1]], axis=-1)
+            get_max = np.nanmax if nan_policy == 'omit' else np.max
+            reference = get_max(input_data[..., limits[0]:limits[1]],
+                                axis=-1)
```

Comment:
Why not directly use `np.nanmax(..., nan_policy=nan_policy)`?
Might also be changed in the elif case below.